src/main/java/org/ow2/proactive/workflow_catalog/rest/controller/WorkflowRevisionController.java

```diff
@@ -125,4 +126,13 @@ public PagedResources list(@PathVariable Long bucketId,
                 bucketId, Optional.ofNullable(workflowId), query, pageable, assembler);
     }
 
+    @ApiOperation(value = "Delete a workflow's revision", notes = "Returns the deleted WorkflowRevision metadata. If the latest revision is deleted, the previous one becomes the new latest revision.")
```

Comment:
The following explanation looks misleading to me:

> If the latest revision is deleted, the previous one becomes the new latest revision. 

I think the explanation should consider two cases:

  - What happen in a general case where the revision to remove is not the last one? (i.e. what you have explained)
  - What happen when a workflow contains a single revision and this last is removed? (i.e. does the whole workflow is removed?)